src/aui-toolbar/js/aui-toolbar.js

```diff
@@ -389,7 +392,12 @@ ToolbarRenderer.prototype = {
             }
 
             // Add cssClass support
-            cssClass = [CSS_BTN, CSS_BTN_DEFAULT, value.cssClass];
+            cssClass = [CSS_BTN_TOOLBAR, value.cssClass];
+
+            if (value.addDefaultButtonCssClasses !== false) {
```

Comment:
It is weird set a condition where you compare with a boolean, do you really need this?